index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE HTML>
+<html lang="en">
+<head>
+	<title>Jordan Addis - Portfolio</title>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<!-- FavIcon -->
+	<link rel="apple-touch-icon" sizes="180x180" href="apple-touch-icon.png">
+	<link rel="icon" type="image/png" sizes="32x32" href="favicon-32x32.png">
+	<link rel="icon" type="image/png" sizes="16x16" href="favicon-16x16.png">
+	<!-- Styling -->
+	<link href="normalize.css" type="text/css" rel="stylesheet">
+	<link href="style.css" type="text/css" rel="stylesheet">
+	<!-- Fonts -->
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Space+Mono&family=Work+Sans:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet"> 
+	<!-- Scripts -->
+	<script src="https://kit.fontawesome.com/7c6af540e7.js" crossorigin="anonymous"></script>
+</head>
+<body>
+	<a class="anchor" id="top"></a>
+	<nav>
+		<div class="navLeft">
+			<a href="#" class="navLogo">
+				<h3>Jordan Addis</h3>
+			</a>
+		</div>
+		<div class="navRight">
+			<a href="#top" class="navLink">Home</a>
+			<a href="#about" class="navLink">About Me</a>
+			<a href="#projects" class="navLink">Projects</a>
+			<a href="#contact" class="navLink">Contact</a>
+		</div>
+		<a href="#" class="burgerMenu">
+				<div></div>
+				<div></div>
+				<div></div>
+		</a>
+	</nav>
+	<header>
+		<div class="headerContent">
```

Comment:
extra layer of div is potentially not needed - layout can be achieved without it

Reply:
Yeah, have moved the flex properties from headerContent onto header and have removed div.